src/openscad.cc

```diff
@@ -288,33 +288,45 @@ int cmdline(const char *deps_output_file, const std::string &filename, const std
 
 	if(!export_format.empty()) {
 		// If command line option --export-format is specified, use that format.
-		formatName = export_format;
+		// Confirm that it is a valid export format
```

Comment:
This confirms that the extension (suffix) entered in `--export-format` is valid.  There wasn't an early check like this before.
It is not part of the original issue but seems appropriate.